src/CalcManager/CEngine/scicomm.cpp

```diff
@@ -214,7 +212,7 @@ void CCalcEngine::ProcessCommandWorker(OpCode wParam)
             // Here * is m_nPrevOpCode, m_currentVal is 2  (by 1*2), m_nLastCom is +, m_nOpCode is ^
             if (m_fPrecedence && 0 != m_nPrevOpCode)
             {
-                nPrev = NPrecedenceOfOp(m_nPrevOpCode);
+                int nPrev = NPrecedenceOfOp(m_nPrevOpCode);
                 nx = NPrecedenceOfOp(m_nLastCom);
```

Comment:
It is a bit difficult since nx is used in the switch statement that comes later, which is not nested in any other statement, but I will see if that is possible without any drastic changes to the function

Reply:
Update: I managed to do it